13_3_Setting_Up_a_Channel.md

```diff
@@ -0,0 +1,260 @@
+    "commit_hash": "8cb1276dbf0bfd9fcbf599df87a43238e599eaac",
+    "identity_pubkey": "0302d48972ba7eef8b40696102ad114090fd4c146e381f18c7932a2a1d73566f84",
+    "alias": "0302d48972ba7eef8b40",
+    "color": "#3399ff",
+    "num_pending_channels": 0,
+    "num_active_channels": 0,
+    "num_inactive_channels": 0,
+    "num_peers": 0,
+    "block_height": 1780686,
+    "block_hash": "000000000000000beb29fa5d3afb713a253f949b12f5f5be25935bb26764e321",
+    "best_header_timestamp": "1594737322",
+    "synced_to_chain": true,
+    "synced_to_graph": true,
+    "testnet": true,
+    "chains": [
+        {
+            "chain": "bitcoin",
+            "network": "testnet"
+        }
+    ],
+    "uris": [
+    ],
+    "features": {
+        "0": {
+            "name": "data-loss-protect",
+            "is_required": true,
+            "is_known": true
+        },
+        "5": {
+            "name": "upfront-shutdown-script",
+            "is_required": false,
+            "is_known": true
+        },
+        "7": {
+            "name": "gossip-queries",
+            "is_required": false,
+            "is_known": true
+        },
+        "9": {
+            "name": "tlv-onion",
+            "is_required": false,
+            "is_known": true
+        },
+        "13": {
+            "name": "static-remote-key",
+            "is_required": false,
+            "is_known": true
+        },
+        "15": {
+            "name": "payment-addr",
+            "is_required": false,
+            "is_known": true
+        },
+        "17": {
+            "name": "multi-path-payments",
+            "is_required": false,
+            "is_known": true
+        }
+    }
+}
+```       
+
+#### Connect to remote node
+
+The first thing you need to do is connect your node to a peer. This is done with the `lightning-cli connect` command. Remember that if you want more information on this command, you should type `lightning-cli help connect`.   The connect RPC command establishes a new connection with another node in the Lightning Network.
+
+To connect your node to a remote peer you need it's id that represents the target node’s public key. As a convenience, id may be of the form id@host or id@host:port. In this case, the port parameter must be changed due we're running two nodes on the same machine,  c-lightning node it's running on 9735 default port and we've started LND node on 9736 port with this parameter.
+
+```       
+$ lnd --listen=0.0.0.0:9736 
+```       
+We can check it's listening on 9736 port using this command:
+
+```       
+$ netstat -aon | grep 9736
+tcp6       0      0 :::9736                 :::*                    LISTEN      off (0.00/0/0)
+$ 
+```       
+Now we can use `lightning-cli connect` command to reach node as a peer:
+
+```       
+$ lightning-cli --network=testnet connect 0302d48972ba7eef8b40696102ad114090fd4c146e381f18c7932a2a1d73566f84@127.0.0.1:9736
+{
```

Comment:
Need to clarify where 0302d48972ba7eef8b40696102ad114090fd4c146e381f18c7932a2a1d73566f84 comes from. It might be that you move that getinfo command into the second of setting up the LND peer, so that you can have people set it up, then get the info, then come back and enter that into c-lightning.

Overall, we need a clear way to designate which commands are entered on which machine, possibly just:

cl$
lnd$

Reply:
Done,  I've included a paragraph explaining how to get pub key.   I've removed all LND commands.